src/test/java/innohack/gem/entity/rule/rules/TikaContentRegexTest.java

```diff
@@ -17,6 +17,18 @@ public void testValidCheckTikaContent() throws Exception {
     assertTrue(contentRegexCheck.check(pdfFile));
   }
 
+  @Test
+  public void testValidCheckWithoutCaptureGroup() throws Exception {
+
+    GEMFile pdfFile = new GEMFile("story_0.pdf", "src/test/resources/extract");
+    pdfFile.extract();
+    TikaContentRegex contentRegexCheck = new TikaContentRegex("[white|black]");
```

Comment:
This test for the presence of any letters `w, h, i, t, e, |, b, l, a, c, k`. Is that the intention? If not, use a word boundary match?

Reply:
oh thats a mistake. i actually want word boundary or non-capturing group ie `(?:white|back)`